apps/site/src/views/Landing/components/ApplyButton/ApplyButton.module.scss

```diff
@@ -0,0 +1,33 @@
+@use "bootstrap-utils" as utils;
+
+.applyButtonContainer {
+	position: relative;
+}
+
+.applyButton {
+	position: relative;
+	transform: skew(-30deg);
+	height: 80px;
+	width: 250px;
```

Comment:
Nitpick: compared to the Figma, this parallelogram looks a bit too tall.

Reply:
I changed the text from a h2 to a h1 to make it fill the button up more and look closer to the Figma project. Let me know if it looks good!